README.md

```diff
@@ -3,55 +3,76 @@
 ## Install
 
 - This assumes an environment is already set up for npm packages - if not, please use npm init following the steps at [https://docs.npmjs.com/getting-started/using-a-package.json](https://docs.npmjs.com/getting-started/using-a-package.json).
-- run `npm install @patternfly/patternfly-next --save`
+- run `npm install @patternfly/patternfly-next`
```

Comment:
We should be using --save

Reply:
It's important to make sure users' dependency list gets updated to include patternfly-next. For a long time `--save` was the way to do that, and a lot of READMEs still specify `npm install --save`. But since npm v5 `--save` has been default ([ref](https://github.com/npm/npm/releases/tag/v5.0.0)) and there is no `--save` option for the current version of npm (the equivalent is `--save-prod`, and that's the default ([ref](https://docs.npmjs.com/cli/install))). How about something like this?

> - run `npm install @patternfly/patternfly-next` (or, if you're using npm < v5.0.0, `npm install --save @patternfly/patternfly-next`)